README.md

```
@@ -1,96 +1,300 @@
+ 
+  - Docker 
+## 🏗 Deployment Steps  
+
+### **<p id="docker-networking">Deployment Using EC2 Instance</p>**  
+
+#### Step 1: Set up the Master Node (EC2 Instance)
+**Go to AWS your master-node instances and select security groups and go to add rule `8080` save this.**
+**Configure EC2 Security Group:**
++1. Navigate to your EC2 instance's security group
++2. Add inbound rule:
++   - Port: 8080
```

Markdownlint (0.35.0)

Expected: asterisk; Actual: plus
Unordered list style

(MD004, ul-style)